src/style.css

```
@@ -0,0 +1,135 @@
+html {
+  font-family: Roboto, sans-serif;
+  --color-main-text: #060b37;
+  --color-secondary-text: #616070;
+  --color-accent: #00acdc;
+}
+
+body {
+  margin: 0;
+}
+
+h2 {
+  margin: 0;
+}
+
+p {
+  margin: 0;
+}
+
+.product-card {
+  width: 198px;
```

or you can even use  `box-sizing: border-box` at `*` selector
```suggestion
  box-sizing: border-box;
  width: 200px;
```

---

not fixed
add `box-sizing: border-box` to `*` selector, and define `width: 200px`